main_classes.py

```diff
@@ -4,83 +4,62 @@
-        if len(self.botspam) > 0:
-            self.botspam[-1][1] += message
-            return
-        print("Attempted to add message to non-existent bot-spam message!\n{}".format(message))
-        
-    def story(self,message):
-        self.storytime.append(message)
-    def story_add(self,message):
-        if len(self.storytime) > 0:
-            self.storytime[-1] += message
-            return
-        print("Attempted to add message to non-existent story-time message!\n{}".format(message))
+    def log(self,content,temporary = False):
+        self.gamelog.append(Message(content,temporary))
+    def log_add(self,moar_content):
+        self.gamelog[-1].add(moar_content)
     
-    def respond(self,message,channel):
-        self.channel.append([message,channel])
-    def respond_add(self,message):
-        if len(self.channel) > 0:
-            self.channel[-1][0] += message
-            return
-        print("Attempted to add message to non-existent response!\n{}".format(message))
-
-    def dm(self,message,channel):
-        self.player.append([message,channel])
-    def dm_add(self,message):
-        if len(self.player) > 0:
-            self.player[-1][0] += message
-            return
-        print("Attempted to add message to non-existent DM!\n{}".format(message)) 
+    def spam(self,content,temporary = False):
+        self.botspam.append(Message(content,temporary))
+    def spam_add(self,moar_content):
+        self.botspam[-1].add(moar_content)
 
-    # Gain all info from another mailbox and put them inside this one.
-    def suck(self,old_mail):
-        for msg in old_mail.gamelog:
-            self.log(msg)
-        for msg in old_mail.botspam:
-            self.spam(msg)
-        for msg in old_mail.storytime:
-            self.story(msg)
-        for parcel in old_mail.channel:
-            self.channel.append(parcel)
-        for parcel in old_mail.player:
-            self.player.append(parcel)   
-
-# This class contains all information that needs to be global. It is used for retrieving and passing on information from different functions.
-class Game_Control:
+    def story(self,content,temporary = False):
+        self.storytime.append(Message(content,temporary))
+    def story_add(self,moar_content):
+        self.storytime[-1].add(moar_content)
     
-    # All attacks are listed in here, including the role that attacked them.
-    kill_queue = []
-
-    # The time in the game. Either "Day", "Night" or "Undefined"
-    time = "Undefined"
-
-    # This command is executed when the day is started.
-    def start_day(self):
-        self.time = "Day"
+    def msg(self,content,destination,temporary = False):
+        self.channel.append(Message(content,temporary,destination))
+    def msg_add(self,moar_content):
+        self.channel[-1].add(moar_content)
+    
+    def dm(self,content,user_id,temporary = False):
+        self.player.append(Message(content,temporary,user_id))
+    def dm_add(self,moar_content):
```

Comment:
`moar_content`?

(also wtf is this)

Reply:
That is in case a message was incomplete, and another part needs to be added later on. This way, multiple scenarios can be written without having to think about each permutation.